src/UI/GameStatsController.java

```diff
@@ -13,23 +13,26 @@
 public class GameStatsController implements InfoController
 {
   private GameInstance myGame;
-  private ArrayList<InfoPage> infoPages;
-  
-  private OptionSelector coOptionSelector;
+  private ArrayList<ArrayList<InfoPage>> infoPages;
+
   private OptionSelector pageSelector;
+  private int shiftDown = 0;
 
   public GameStatsController( GameInstance game )
   {
     myGame = game;
 
-    infoPages = new ArrayList<InfoPage>();
-    infoPages.add(new InfoPage(InfoPage.PageType.CO_HEADERS));
-    infoPages.add(new InfoPage(InfoPage.PageType.GAME_STATUS));
-    
-    coOptionSelector = new OptionSelector(myGame.commanders.length);
+    infoPages = new ArrayList<ArrayList<InfoPage>>();
+
+    ArrayList<InfoPage> headers = new ArrayList<InfoPage>();
+    headers.add(new InfoPage(InfoPage.PageType.CO_HEADERS));
+    infoPages.add(headers);
+
+    ArrayList<InfoPage> status = new ArrayList<InfoPage>();
+    status.add(new InfoPage(InfoPage.PageType.GAME_STATUS));
+    infoPages.add(status);
```

Comment:
I think we decided it would make sense to break the detailed info into separate pages per CO.